apps/E2E/src/ContextualMenu/pages/ContextualMenuPageObject.win.ts

```diff
@@ -5,44 +5,31 @@ import {
 import { BasePage, By } from '../../common/BasePage';
-
-/* This enum gives the spec file an EASY way to interact with SPECIFIC UI elements on the page.
- * The spec file should import this enum and use it when wanting to interact with different elements on the page. */
-export const enum ContextualMenuSelector {
-  ContextualMenu = 0, //this._primaryComponent
-  ContextualMenuItem, //this._contextualMenuItem
-}
+import { Keys } from '../../common/consts';
 
 class ContextualMenuPageObject extends BasePage {
   /******************************************************************/
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
-  async waitForContextualMenuItemsToOpen(timeout?: number): Promise<void> {
-    await browser.waitUntil(async () => await this.contextualMenuItemDisplayed(), {
-      timeout: timeout ?? this.waitForUiEvent,
-      timeoutMsg: 'The Contextual Menu Items did not open.',
-      interval: 1000,
-    });
+  /* Waits for the MenuItems to be displayed and returns true / false whether that occurs. */
+  async waitForContextualMenuItemsToDisplay(timeout?: number): Promise<boolean> {
+    await this.waitForCondition(async () => this.contextualMenuItemDisplayed(), 'The Contextual Menu Items did not display.', timeout);
+    return await this.contextualMenuItemDisplayed();
   }
 
   /* Whether the contextual menu item is displayed or not. It should be displayed after clicking on the MenuButton */
   async contextualMenuItemDisplayed(): Promise<boolean> {
     return await (await this._contextualMenuItem).isDisplayed();
   }
 
-  /* Sends a Keyboarding command on a specific UI element */
-  async sendKey(contextualMenuSelector: ContextualMenuSelector, key: string): Promise<void> {
-    await (await this.getContextualMenuSelector(contextualMenuSelector)).addValue(key);
-  }
-
-  /* Returns the correct WebDriverIO element from the ContextualMenuSelector string */
-  async getContextualMenuSelector(contextualMenuSelector?: ContextualMenuSelector): Promise<WebdriverIO.Element> {
-    if (contextualMenuSelector == ContextualMenuSelector.ContextualMenu) {
-      return await this._primaryComponent;
-    } else if (contextualMenuSelector == ContextualMenuSelector.ContextualMenuItem) {
-      return await this._contextualMenuItem;
-    } else {
-      return await this._primaryComponent;
+  /* Closes the ContextualMenu by sending an ESCAPE key input to the menu button */
+  async closeContextualMenu(): Promise<void> {
+    if (await this.contextualMenuItemDisplayed()) {
+      await this.sendKeys(this._contextualMenuItem, [Keys.ESCAPE]);
+      await this.waitForCondition(
+        async () => !(await this.contextualMenuItemDisplayed()),
+        "Pressed 'ESC' on the ContextualMenu button, but the ContextualMenu items are still displayed - the ContextMenu failed to close.",
```

Comment:
Fix this message to "Pressed 'ESC' on _contextual menu item_, but the _contextual menu items_ are still displayed."  

I also liked how you worded [this error message for Menu](https://github.com/microsoft/fluentui-react-native/pull/2447/files#diff-09e2d8b1547625fcc5407ba70eadb2c9cc698b80ade10152c25d34d9d44b6fc3R181). We could do the same here by saying "The contextual menu appears to still be open." but it's up you.